### ISSUE TITLE
shibboleth config needs upgrading

### ISSUE THREAD

**Issue by mwest1066** [PrairieLearn/PrairieLearn#1266]
Our logs have lots of this in them:

```
Sep 19 01:40:37 ip-10-0-0-204.us-east-2.compute.internal shibboleth[33439]: WARN Shibboleth.Config : DEPRECATED: legacy 2.0 configuration, support will be removed from a future version of the software
Sep 19 01:40:37 ip-10-0-0-204.us-east-2.compute.internal shibboleth[33439]: WARN Shibboleth.RequestMapper : DEPRECATED: legacy 2.0 configuration, support will be removed from a future version of the software
Sep 19 01:40:37 ip-10-0-0-204.us-east-2.compute.internal shibboleth[33439]: ERROR Shibboleth.Application : SSO discoveryProtocol specified without discoveryURL
```

**Comment by trombonekenny** [comment]
I'll fix this the next time I'm in ansible. I know I already fixed it for another host -- cbtf? pl-dev?

**Comment by mwest1066** [comment]
Awesome! Let me know if you need any help from me.

**Comment by trombonekenny** [comment]
I'll look at this on 11/19 or 11/20.

**Comment by trombonekenny** [comment]
Addressed in pl-ansible PR#44.

**Comment by nwalters512** [comment]
We're moving away from doing Shibboleth via Apache, so I believe there's no longer a need for this work. @trombonekenny please reopen if you think we should still do this!
